extensions/s3/src/test/java/com/hazelcast/jet/s3/S3SinkTest.java

```diff
@@ -51,6 +54,24 @@ public void deleteObjects() {
         if (!identifiers.isEmpty()) {
             client.deleteObjects(b -> b.bucket(bucketName).delete(d -> d.objects(identifiers)));
         }
+
+        int sleepMillis = (int) (SECONDS.toMillis(WAIT_AFTER_CLEANUP_IN_SECS) / 10);
+        long deadline = System.currentTimeMillis() + SECONDS.toMillis(WAIT_AFTER_CLEANUP_IN_SECS);
+        int keyCount;
+        while ((keyCount = client.listObjectsV2(ListObjectsV2Request
```

Comment:
we can move this to a method for readibility and for re-use:
`client.listObjectsV2(ListObjectsV2Request.builder().bucket(bucketName).build())`